src/components/Organisms/ToDoCard/index.jsx

```diff
@@ -34,6 +36,7 @@ export const ToDoCard = () => {
     const modifiedTaskList = [...taskList];
     if (value === "") {
       modifiedTaskList.splice(index, 1);
+      AlertHandlerContext.setAlert("タスクの名前が設定されていません。");
```

Comment:
```suggestion
     const filteredOnTaskNameChange = taskList.filter(
        (_, idx) => idx !== index
      );
      setTaskList(filteredOnTaskNameChange);
      AlertHandlerContext.setAlert("タスクの名前が設定されていません。");
```
文字を何も入力しないとき、写真のようになっていたので修正をお願いします。
![スクリーンショット (56)](https://github.com/Yuseidosukoiya/Yusei-frontend-basic-2022-todoapp-2/assets/137466167/0d610a3a-2624-4ef8-87fd-315b551623f6)

filterの使い方は、以下の参考サイトを見てください！
https://qiita.com/itachi/items/4184b2afc35b55b45568
https://jsnotice.com/posts/2020-07-19/index.html